project/Dependencies.scala

```diff
@@ -96,6 +97,7 @@ object Dependencies {
     Compile.akkaHttp2Support,
     Compile.akkaDiscovery,
     // these two are available when used through Play, which is also the only case when they are needed
+    Compile.akkaSlf4j % "provided",
```

Comment:
The comment above seems to refer to:

```scala
    Compile.play % "provided",
    Compile.playAkkaHttpServer % "provided",
```
The comment should state `these three` or be moved one line down.

Reply:
moved one line down